WordPress/Classes/ViewRelated/Stats/JetpackLoginViewController.swift

```diff
@@ -181,14 +190,36 @@ class JetpackLoginViewController: UIViewController {
         WPAnalytics.track(stat, withProperties: properties)
     }
 
+    private func openWebView(for webviewType: JetpackWebviewType) {
+        guard let url = webviewType.url else {
+            return
+        }
+
+        let webviewViewController = WebViewControllerFactory.controller(url: url)
+        let navigationViewController = UINavigationController(rootViewController: webviewViewController)
+        present(navigationViewController, animated: true, completion: nil)
+    }
+
     // MARK: - Actions
 
     @IBAction func didTouchSignInButton(_ sender: Any) {
         signIn()
     }
 
     @IBAction func didTouchInstallJetpackButton(_ sender: Any) {
-        openInstallJetpackURL()
+        guard Feature.enabled(.jetpackRemoteInstallation) else {
+            openInstallJetpackURL()
+            return
+        }
+        // Open new Jetpack remote installation
```

Comment:
I know this is a work in progress, but shall we leave the button functional while the feature flag is enabled, until you get to implementing the new flow?